src/main/target/SITL/target.c

```diff
@@ -60,15 +60,22 @@ const timerHardware_t timerHardware[1]; // unused
 uint32_t SystemCoreClock;
 
 static fdm_packet fdmPkt;
+static rc_packet rcPkt;
 static servo_packet pwmPkt;
+static servo_packet_raw pwmRawPkt;
```

Comment:
Did you check that mutex locking? I did not analyze code carefully, but there may be races ...

Reply:
Add mutex protection.